packages/react/package.json

```diff
@@ -68,6 +68,7 @@
     "@typescript-eslint/eslint-plugin": "5.10.2",
     "@wessberg/rollup-plugin-ts": "2.0.4",
     "babel-jest": "^26.0.1",
+    "babel-plugin-polyfill-corejs2": "^0.4.1",
```

Comment:
I don't think this is needed in this PR? It looks like it could be an improvement but maybe in a separate PR where it's implemented and tested.

Reply:
Sure! I had some problems to start the project without adding this plugin, but I ran the builds again just now and everything seems to work fine now. Not sure what caused the issue then, but I'll remove this as unnecessary 🙂